sparrow_datums/keypoints/keypoints.py

```diff
@@ -0,0 +1,183 @@
+from typing import TypeVar
+
+import numpy as np
+
+from ..chunk import Chunk
+from ..exceptions import ValidationError
+from ..types import FloatArray, PType
+
+T = TypeVar("T", bound="Keypoints")
+
+# Note to self: np.view is making a shallow copy of an array.
+class Keypoints(Chunk):
+    """Dense data arrays for keypoints.
+
+    Parameters
+    ----------
+    Chunk : Chunk
+        Base class for dense data arrays with metadata.
```

Comment:
The Parameters section isn't quite right -- `Keypoints` takes the same args as `Chunk` [here](https://github.com/sparrowml/sparrow-datums/blob/27bb011cd1fd20165ecfd55fab91d728d68ad4e1/sparrow_datums/chunk.py#L22). Since they're all defined on the parent class, it should be safe to delete this section.